doc/source/ray-air/examples/batch_tuning_ds.ipynb

```diff
@@ -0,0 +1,1479 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Batch Training with Ray Datasets"
```

Comment:
let's not move this file here

Reply:
oops, accidentally included in my push...  OK to remove it.  Thanks.